bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -44,23 +44,23 @@
     protected String projectId;
     protected String zoneId;
     protected String clusterId;
-    protected Map<String, String> additionalConfiguration = new HashMap<>();
+    protected Map<String, String> additionalConfigurations = new HashMap<>();
```

Comment:
I think singular case is fine here.  An XML file, for example, is the "configuration" not the "configurations"